keymanapp/keyman
# chore(android/engine): Fix dictionary selector and back arrow styling

Follow-on to #4662 and fixes #4715

This fixes the Dictionary Settings menu selector to have a transparent instead of gray background.

Also fixes the back arrow button to be black instead of gray in the titlebars. Note, the Keyman "Settings" menu uses a different layout, so that's not updated.

## User Testing
Setup: Install the test app

* **TEST_STYLING** Test the styling on dictionary settings menu
1. Load the Keyman app
2. Go to the "Settings" menu --> Installed Languages --> English
3. Verify on the "English Settings" menu that the back arrow is styled black
![Screenshot_1630554169](https://user-images.githubusercontent.com/7358010/131785890-802c1f07-249c-451d-b84f-4ad82444d46d.png)

4. Click the bottom menu "Dictionary" to access Dictionary Settings
5. On the "Dictionary Settings" menu, hold on "English dictionary (MTNT)"
6. Verify the entire selection has the same color styling (checkbox and  forward arrow don't have a gray background)
![dictionary settings](https://user-images.githubusercontent.com/7358010/131779289-b3bf673e-5551-4531-928a-48f192bfe2f1.png)
7. Click around other menus and verify other styling is consistent.
